comics/aggregator/crawler.py

```diff
@@ -1,251 +1,252 @@
+            self.feed = FeedParser(feed_url)
+        return self.feed
+
+    def parse_page(self, page_url):
+        if page_url not in self.pages:
+            self.pages[page_url] = LxmlParser(page_url, headers=self.headers)
+        return self.pages[page_url]
+
+    def string_to_date(self, *args, **kwargs):
+        return datetime.datetime.strptime(*args, **kwargs).date()
+
+    def date_to_epoch(self, date):
+        """The UNIX time of midnight at ``date`` in the comic's time zone"""
+        naive_midnight = datetime.datetime(date.year, date.month, date.day)
+        local_midnight = pytz.timezone(self.time_zone).localize(naive_midnight)
+        return int(time.mktime(local_midnight.utctimetuple()))
+
+
+class KingFeaturesCrawlerBase(CrawlerBase):
+    """Base comic crawler for King Features comics"""
+
+    def crawl_helper(self, domain, pub_date):
+        date = pub_date.strftime('%B-')
+        date += pub_date.strftime('%d').lstrip('0')
+        date += pub_date.strftime('-%Y')
+        page_url = 'http://%s/comics/%s' % (domain, date)
+        page = self.parse_page(page_url)
+        url = page.src('img[src*="safr.kingfeatures.com"]')
+        return CrawlerImage(url)
+
+
+class GoComicsComCrawlerBase(CrawlerBase):
+    """Base comic crawler for all comics hosted at gocomics.com"""
+
+    # It doesn't want us getting comics because of a User-Agent check.
+    # Look! I'm a nice, normal Internet Explorer machine!
+    headers = {
+        'User-Agent': (
+            'Mozilla/4.0 (compatible; MSIE 8.0; Windows NT 5.1; '
+            'Trident/4.0; .NET CLR 1.1.4322; .NET CLR 2.0.50727; '
+            '.NET CLR 3.0.4506.2152; .NET CLR 3.5.30729'),
+    }
+
+    def crawl_helper(self, url_name, pub_date):
+        page_url = 'http://www.gocomics.com/%s/%s' % (
+            url_name, pub_date.strftime('%Y/%m/%d/'))
+        page = self.parse_page(page_url)
+        page.remove('.feature_item')
+        url = page.src('img.strip')
+        return CrawlerImage(url)
+
+
+class PondusNoCrawlerBase(CrawlerBase):
+    """Base comics crawler for all comics posted at pondus.no"""
+
+    time_zone = 'Europe/Oslo'
+
+    def crawl_helper(self, url_id):
+        page_url = 'http://www.pondus.no/?section=artikkel&id=%s' % url_id
+        page = self.parse_page(page_url)
+        url = page.src('.imagegallery img')
+        return CrawlerImage(url)
+
+
+class HeltNormaltCrawlerBase(CrawlerBase):
+    """Base comics crawler for all comics posted at heltnormalt.no"""
+
+    headers = {'User-Agent': 'Mozilla/5.0'}
+    time_zone = 'Europe/Oslo'
+
+    def crawl_helper(self, short_name, pub_date):
+        date_string = pub_date.strftime('%Y/%m/%d')
+        short_name2 = short_name.replace('_', '')
+        page_url = 'http://heltnormalt.no/%s/%s' % (short_name2, date_string)
+        page = self.parse_page(page_url)
+        url = page.src('img[src*="/img/%s/%s"]' % (short_name, date_string))
+        return CrawlerImage(url)

```

Comment:
I think you converted the file endings to Windows format or something as the whole file has changed.